atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchEventCacheIntegrationTest.java

```diff
@@ -129,22 +127,19 @@ boolean isDev() {
     private LockWatchEventCache eventCache;
     private int part;
 
-    @Rule
-    public TestName name = new TestName();
-
-    @Before
+    @BeforeEach
     public void before() {
         createEventCache(5);
         part = 1;
     }
 
-    private void verifyStage() {
+    private void verifyStage(TestInfo testInfo) {
         ObjectMapper mapper = new ObjectMapper()
                 .enable(SerializationFeature.INDENT_OUTPUT)
                 .registerModule(new Jdk8Module())
                 .registerModule(new GuavaModule());
         try {
-            Path path = Paths.get(BASE + name.getMethodName() + "/event-cache-" + part + ".json");
+            Path path = Paths.get(BASE + testInfo.getTestMethod().get().getName() + "/event-cache-" + part + ".json");
```

Comment:
I don't expect it to be empty ever. However, maybe it is better to add a check anyway?